Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -565,11 +565,13 @@ extension ProductStore {
 ///
 /// - duplicatedSKU: the SKU is used by another Product.
 /// - invalidSKU: the SKU is invalid or duplicated.
+/// - password: the password of a product cannot be updated.
 /// - unknown: other error cases.
 ///
 public enum ProductUpdateError: Error {
     case duplicatedSKU
     case invalidSKU
+    case password
```

Comment:
It would be nice if the `case` itself is descriptive too. 🙂 

```suggestion
    case passwordCannotBeUpdated
```

Reply:
updated in: https://github.com/woocommerce/woocommerce-ios/pull/2785/commits/282dfd9948a658854eefeb749d2be10a491d5362